weblate/accounts/tests/test_registration.py

```diff
@@ -531,13 +531,13 @@ def test_github(self, confirm=None, fail=False):
             orig_backends = social_django.utils.BACKENDS
             social_django.utils.BACKENDS = GH_BACKENDS
 
-            httpretty.register_uri(
-                httpretty.POST,
+            responses.register_uri(
```

Comment:
```suggestion
            responses.add(
```

This method is called differently (and it repeats in basically every test).

Reply:
@nijel Do you know what to do about "has.response"?

Reply:
No, I didn't yet use much of this library.